Cargo.toml

```diff
@@ -232,7 +232,7 @@ linkify = "0.10.0"
 log = { version = "0.4.16", features = ["kv_unstable_serde"] }
 ordered-float = "2.1.1"
 palette = { version = "0.7.5", default-features = false, features = ["std"] }
-parking_lot = "0.11.1"
+parking_lot = "0.12.1"
```

Comment:
We have 2 versions of parking_lot in our .lock file (and `0.12.1` seems to be brought in by transitive dep) so I've bumped that one up.